File: alter-textfile/solve-txt.js
import * as fs from 'fs'

fs.readFile('sample.txt', 'utf-8', (err, data) => {
    if (err) {
        console.error(err)
        return
    }
    const modifiedData = data.replace(/\n/g, ' ')
    console.log(modifiedData)
})

File: alter-textfile/solve-txt.go
package main

import (
	"fmt"
	"os"
	"log"
	"strings"
)

func main()  {
	data, err := os.ReadFile("sample.txt")
	if err != nil {
		log.Fatal(err)
	}

	modifiedData := strings.ReplaceAll(string(data), "\n", " ")
	fmt.Println(modifiedData)
}